nlp/front/storage-mongo/target/generated-sources/kapt/compile/ai/tock/nlp/front/shared/config/FaqSettings_Deserializer.kt

```diff
@@ -18,7 +18,7 @@ import org.litote.jackson.JacksonModuleServiceLoader
 import org.litote.kmongo.Id
 
 internal class FaqSettings_Deserializer : JsonDeserializer<FaqSettings>(),
```

Comment:
You don't need to modify this file for this feature. It's better to do a rollback.